exporter/app/core/config.py

```diff
@@ -6,16 +6,27 @@
 BUILD_INFO_REQUEST='/api/json?tree=result'
 NODES_INFO_REQUEST = '/computer/api/json?tree=computer[displayName,offline,assignedLabels[name],executors[idle]]'
 
+JOBS_REQUEST = 'api/json?depth=1&tree=jobs[name,labelExpression,url]'
+LABEL_IN_JOB = 'api/json?tree=labelExpression'
 
 db_username = os.getenv('DB_USERNAME')
 db_password = os.getenv('DB_PASSWORD')
 db_host = os.getenv('DB_HOST')
 db_port = os.getenv('DB_PORT')
 
-
 jenkins_url = os.getenv('JENKINS_URL')
 username = os.getenv('USERNAME')
 token = os.getenv('TOKEN')
 port = int(os.getenv('PORT'))
 
+collect_jobs_labels = os.getenv('COLLECT_JOBS_LABELS', "")
```

Comment:
Так будет сохранено булево значение, вместо строки и им будет проще пользоваться
```suggestion
collect_jobs_labels = os.getenv('COLLECT_JOBS_LABELS', "") == "True"
```